<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QWidget" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>614</height>
   </rect>
  </property>
  <widget class="QLabel" name="labelBackground">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>9</y>
     <width>641</width>
     <height>607</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelOrc">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelTooltip">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>480</width>
     <height>480</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRefresh">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>230</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">.QPushButton{
background-color: rgba(255, 255, 255, 0.6);
border:none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="labelStrTip">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>450</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../../resource/sVanilla.qrc"/>
 </resources>
 <connections/>
</ui>
